x-pack/plugins/security/server/config.ts

```diff
@@ -391,11 +391,18 @@ export function createConfig(
 function getSessionConfig(session: RawConfigType['session'], providers: ProvidersConfigType) {
   return {
     cleanupInterval: session.cleanupInterval,
-    getExpirationTimeouts({ type, name }: AuthenticationProvider) {
+    getExpirationTimeouts(provider?: AuthenticationProvider) {
```

Comment:
I think we're missing unit tests for this change in `config.test.ts`. The existing tests all assume that this parameter will be defined